pyproject.toml

```diff
@@ -66,9 +66,10 @@ exclude = '''
     | buck-out
     | build
     | dist
+    | migrations
```

Comment:
I am not seeing this as a standard in many other repos.

Reply:
Migrations are dynamically generated and should be out of scope of formatters IMHO. This has bitten my team in the past where a find & replace hit a previous migration then was swallowed by applying black. That issue made it to production code.

Reply:
I would prefer to stay in line with dev-standards. If it is an issue, that would seem the place to bring up.

Reply:
```suggestion
```

Sounds good. I will post this to the channel for discussion async from this PR :-)

Reply:
agreed, thanks.